iptables.yaml

```diff
@@ -70,6 +71,18 @@ pipeline:
       install -D -m644 ebtables.confd "${{targets.destdir}}"/etc/conf.d/ebtables
 
 subpackages:
+  - name: iptables-privileged
+    description: iptables with extra file capabilities on executables in /sbin only to be used with linkerd.
+    options:
+      no-provides: true
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/sbin
+          cp ${{targets.destdir}}/sbin/xtables-legacy-multi ${{targets.contextdir}}/sbin/xtables-legacy-multi
+          cp ${{targets.destdir}}/sbin/xtables-nft-multi ${{targets.contextdir}}/sbin/xtables-nft-multi
+          setcap cap_net_raw,cap_net_admin+eip ${{targets.contextdir}}/sbin/xtables-legacy-multi
+          setcap cap_net_raw,cap_net_admin+eip ${{targets.contextdir}}/sbin/xtables-nft-multi
```

Comment:
We likely need more than just xtables here so we'd probably need to do something like this:

```
cp -r ${{targets.destdir}} ${{targets.contextdir}}
setcap cap_net_raw,cap_net_admin+eip ${{targets.contextdir}}/sbin/xtables-legacy-multi
setcap cap_net_raw,cap_net_admin+eip ${{targets.contextdir}}/sbin/xtables-nft-multi
```